fedimint-client/src/lib.rs

```diff
@@ -1491,7 +1491,7 @@ impl Client {
             .wait_for(|in_progress| {
                 !in_progress
```

Comment:
this is hard, my head hurts
```suggestion
                in_progress
```